spec/asyncapi.md

```diff
@@ -1064,6 +1064,7 @@ Describes a message received on a given channel and operation.
 
 Field Name | Type | Description
 ---|:---:|---
+<a name="messageObjectMessageId"></a>messageId | `string` | Unique string used to identify the message. The id MUST be unique among all messages described in the API. The messageId value is **case-sensitive**. Tools and libraries MAY use the messageId to uniquely identify a message, therefore, it is RECOMMENDED to follow common programming naming conventions.
```

Comment:
```suggestion
<a name="messageObjectMessageId"></a>messageId | `string` | Unique string used to identify the message. The id MUST be unique among all messages described in the document. The messageId value is **case-sensitive**. Tools and libraries MAY use the messageId to uniquely identify a message, therefore, it is RECOMMENDED to follow common programming naming conventions.
```

I would not mention API since it's not defined anywhere in the spec.

Reply:
I copied it from the `operationId` field.
https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#fixed-fields-7

Reply:
Damn it! Then we need to change it there too. Anyway, not for this PR. Thanks!

Reply:
Yup. Better to keep it consistent for now, and fix it later on in all places.